### PR TITLE
PLANNER-2595 Disable native CI

### PR DESCRIPTION
**Pull request by triceo**

### CODE REVIEW

**Comment by triceo** [comment]
@radtriste Do I need to do anything else to make this happen across the entire KIE CI?

**Comment by triceo** [comment]
@radtriste @Ginxo Any advice?

**Comment by radtriste** [comment]
@triceo This should be enough to remove the job from Jenkins

You need also to remove those 2 lines (or comment them): https://github.com/kiegroup/optaweb-employee-rostering/blob/main/.github/pull_request_template.md?plain=1#L47-L48

**Comment by triceo** [comment]
Thank you, @radtriste !

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -44,6 +44,6 @@ How to retest this PR or trigger a specific build:
 * for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
 * for a <b>Quarkus LTS check</b> please add comment: <b>Jenkins run LTS</b>
 * for a <b>specific Quarkus LTS check</b> please add comment: <b>Jenkins (re)run [optaweb-employee-rostering] LTS</b>
-* for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
-* for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaweb-employee-rostering] native</b>
+# * for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
+# * for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaweb-employee-rostering] native</b>
```
Comment (radtriste) [inline .github/pull_request_template.md:48]:
```suggestion
<!-- 
* for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
* for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaweb-employee-rostering] native</b> 
 -->
```

See  https://gist.github.com/jonikarppinen/47dc8c1d7ab7e911f4c9 :)

**Review by yurloc — APPROVED** [review]
LGTM.

**Pull request merged**